packages/test-studio/schemas/schema.js

```diff
@@ -60,6 +65,7 @@ export default createSchema({
     recursiveArray,
     myObject,
     codeInputType,
+    colorInputType,
```

Comment:
This shouldn't be necessary when the color input implements the schema type part